src/classes/AuthenticationSession.ts

```diff
@@ -32,6 +42,59 @@ export class AuthenticationSession {
     this.promptForPassword();
   }
 
+  public obtainNewAuthTokenUsingRefreshToken(): void {
+    if (!AuthenticationSession.sftpFusionAuthClientId) {
+      logger.error(
+        'Cannot obtain new access token without sftp client id.',
+      );
+      return;
+    }
+
+    if (!AuthenticationSession.sftpFusionAuthClientSecret) {
+      logger.error(
+        'Cannot obtain new access token without sftp client secret.',
+      );
+      return;
+    }
+
+    this.fusionAuthClient.exchangeRefreshTokenForAccessToken(
+      this.refreshToken,
```

Comment:
Upon hitting the endpoint directly with `rested` (postman-like tool) I realize that these parameters are required anyways. At least, the `clientId` and `clientSecret` for this use-case.

So the "issue" with the  fusionauth sdk is both poor documentation and poor use of typing but yay it works~